crates/astria-sequencer-relayer/src/data_availability.rs

```diff
@@ -183,7 +183,7 @@ impl RollupNamespaceData {
     pub fn verify_inclusion_proof(&self, root_hash: [u8; 32]) -> eyre::Result<()> {
         let rollup_data_tree = MerkleTree::from_leaves(self.rollup_txs.clone());
         let rollup_data_root = rollup_data_tree.root();
-        let mut leaf = self.chain_id.as_ref().to_vec();
+        let mut leaf = utils::sha256_hash(self.chain_id.as_bytes()).to_vec();
```

Comment:
does this PR change the format of the `sequence::Action` merkle tree? can you update the doc in `specs/sequencer-inclusion-proofs.md` accordingly?